source/mainnet/net/references/grpc.rst

```diff
@@ -514,21 +535,21 @@ Networks and peers
    :returns: Whether the request was processed successfully.
    :rtype: Bool
 
-.. function:: BanNode(peerElement) -> Bool
+.. function:: BanNode(peer_element: PeerElement) -> Bool
```

Comment:
This seems inconsistent with other cases, where the message type (PeerElement) is expanded into its fields. (Also UnbanNode.)